src/Microsoft.Restier.WebApi/HttpConfigurationExtensions.cs

```diff
@@ -65,6 +72,11 @@ public static class HttpConfigurationExtensions
 
                 var route = config.MapODataServiceRoute(
                     routeName, routePrefix, model, new DefaultODataPathHandler(), conventions, batchHandler);
+
+                // Customized converter should be added in ConfigureApi as service
+                var converter = api.Context.GetApiService<ODataPayloadValueConverter>();
```

Comment:
If the converter will always be retrieved from IEdmModel later, we can simply add
`if (converter == null) converter = new RestierPalyloadValueConverter();`
and omit the default registration.